.github/workflows/database-backup.yml

```diff
@@ -0,0 +1,86 @@
+name: Backup Database to Azure Storage
+concurrency: build_and_deploy_main
+
+on:
+  workflow_dispatch:
+  schedule: # 03:00 UTC
+    - cron: '0 3 * * *'
+
+jobs:
+  backup:
+    name: Backup AKS Database (production)
+    if: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.overwriteThisMorningsBackup == 'true') }}
+    runs-on: ubuntu-latest
+    environment:
+        name: production
+    services:
+      postgres:
+        image: postgres:11.10
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+        - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
```

Comment:
I don't believe you need the postgres service, as you are installing the client and we aren't doing any local db work (like sanitise)

Reply:
removed this part